cli/tests/unit/cron_test.ts

```diff
@@ -172,19 +172,23 @@ Deno.test(async function multipleCrons() {
 
   let count0 = 0;
   let count1 = 0;
-  const promise0 = deferred();
-  const promise1 = deferred();
+  const { promise: promise0, resolve: resolve0 } = Promise.withResolvers<
```

Comment:
Nit/bikeshed: stylistically it might be better to do const promise0 = Promise.withResolvers(...) and then `promise0.resolve` and `await promise0.promise`.